src/components/ThirdPartyEducation.js

```diff
@@ -1,13 +1,73 @@
-import React from "react";
-import { Card } from "react-bootstrap"
-
-export const ThirdPartyEducation = () => {
-
-    return (
-        <Card>
-            <Card.Header> Third Party Education</Card.Header>
-            <Card.Body>
-            </Card.Body>
-        </Card>
-    )
-}
\ No newline at end of file
+import React from "react";
+import { Card } from "react-bootstrap" // npm i bootstrap react-bootstrap
+
+export const ThirdPartyEducation = () => {
+
+
+
+    function retrieve(e){
+        e.preventDefault()
+
+        const searchFrom = document.querySelector('search');
+        const input = document.querySelector('.input');
+        const newsList = document.querySelector('.news-list');
+
+        if (input.value === ''){
+            alert('Input field is empty.')
+            return
+        }
+      
+        newsList.innerHTML = ''
+      
+        const apiKey = '8577c03e48d246b3b0ca7329456e3e84'

```

Comment:
The API key should not be included in source control. I mentioned this to all other teams during the Sprint 2 presentation. 

Once the key is removed from source control, it should be rotated to ensure that it cannot be used maliciously.